WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -4,48 +4,53 @@
   <% if !@competition.registration_opened? %>
     <% if @competition.registration_not_yet_opened? %>
       <div class="alert alert-info">
-        Registration will open in <strong><%= distance_of_time_in_words_to_now(@competition.registration_open) %></strong> on <%= wca_local_time(@competition.registration_open) %>.
+        <% #This internationally works because datetime.distance_in_words.x_days is translated.%>
+        <%= t '.registration.will_open_html',
+              days:distance_of_time_in_words_to_now(@competition.registration_open),
```

Comment:
nitpick: We've always been putting no space before our colons, and 1 space after our colons.